<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>确认密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditConfirmPassword">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>邮箱：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditEmail">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>验证码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCode">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSendCode">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>发送验证码</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegister">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
